src/pages/GithubReposPage/GIthubReposPage.tsx

```diff
@@ -0,0 +1,38 @@
+import StarIcon from '@components/StarIcon';
+import RepoTile from './components/RepoTile';
+import { RepoItem } from '../../store/GithubStore/types';
+import { ReactChild, ReactFragment, useMemo, useState } from 'react';
+import React from 'react';
+import RepoBranchesDrawer from '@components/RepoBranchesDrawer';
+import { EventType } from '@testing-library/react';
+import { Button } from 'antd';
+
+type RepoPage = {
+    tile: any,
+    avatar: string,
+    children: ReactChild,
+}
+
+const GithubReposPage: React.FC<RepoPage> = (props) => {
```

Comment:
если передать так пропсы, не придется в дальнейшим доставать значения через props.tile, например, можно будет сразу обращаться через tile
```suggestion
const GithubReposPage: React.FC<RepoPage> = ({tile, avatar}: RepoPage) => {
```